certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -349,6 +349,116 @@
         },
         "x-ms-trigger": "single"
       }
+    },
+    "/api/v1/webhooks/user/actions/azure-logic-apps": {
+      "get": {
+        "description": "CyberProof Webhooks User Actions List.",
+        "summary": "CyberProof Webhooks User Actions List",
+        "operationId": "CPGetWebhooksActions",
+        "responses": {
+          "200": {
+            "description": "List of Webhooks Actions",
+            "schema": {}
```

Comment:
@Avibek Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**

Reply:
@sriyen-msft But this is internal operation.
So makers won't be exposed to it.
We discussed this in our [last PR](https://github.com/microsoft/PowerPlatformConnectors/pull/1617#discussion_r867969814)